job/job.go

```diff
@@ -43,6 +43,10 @@ type Job struct {
 	// NewJob should be a function on Aggregation
 	AggrID string `json:"aggr_id"`
 
+	// MaxRetries specifies the maximum number for retry attempts. It overrides
+	// the default specified in processor.MaxDownloadRetries
+	MaxRetries NullableInt `json:"-"`
```

Comment:
I don't understand why we need a separate class just to handle the case of an optional int. And I don't understand why there is not `AttrLimit` here as well.